README.md

```diff
@@ -47,16 +47,18 @@ The following workflow is for a new **Helm** operator:
 
 ## Quick Start
 
-First, checkout and install the operator-sdk CLI:
+### Install the Operator SDK CLI
+
+#### Linux
+
+```sh
+curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-linux-gnu && chmod +x operator-sdk && sudo cp operator-sdk /usr/local/bin/ && rm operatr-sdk
```

Comment:
```suggestion
curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-linux-gnu && chmod +x operator-sdk && sudo cp operator-sdk /usr/local/bin/ && rm operator-sdk
```

It doesn't seem possible at the moment, but it would be nice to use the [latest release link](https://help.github.com/en/articles/linking-to-releases#linking-to-the-latest-release) so that we don't have to update the install doc every time we make a release.

To use that, I think we would need to stop using the "This is a pre-release" checkbox when we create a new release, and we would need to stop including the version number in the release binaries that we post to the releases page. WDYT?

Reply:
Yes, but for now we can do this, maybe you can open an issue to discuss it there?

Reply:
Sounds good. Created https://github.com/operator-framework/operator-sdk/issues/1398.